src/main/scala/com/microsoft/ml/spark/cognitive/TextAnalyticsSDK.scala

```diff
@@ -1,289 +1,301 @@
-
-    def getConfidenceScore(score: SentimentConfidenceScores): SentimentConfidenceScoreV4 = {
-      SentimentConfidenceScoreV4(
-        score.getNegative,
-        score.getNeutral,
-        score.getPositive)
-    }
-
-    def getTarget(target: TargetSentiment): TargetV4 = {
-      TargetV4(
-        target.getText,
-        target.getSentiment.toString,
-        getConfidenceScore(target.getConfidenceScores),
-        target.getOffset,
-        target.getLength)
-    }
-
-    def getAssessment(assess: AssessmentSentiment): AssessmentV4 = {
-      AssessmentV4(
-        assess.getText,
-        assess.getSentiment.toString,
-        getConfidenceScore(assess.getConfidenceScores),
-        assess.isNegated,
-        assess.getOffset,
-        assess.getLength)
-    }
-
-    def getSentenceSentiment(sentencesent: SentenceSentiment): SentimentSentenceV4 = {
-      SentimentSentenceV4(
-        sentencesent.getText,
-        sentencesent.getSentiment.toString,
-        getConfidenceScore(sentencesent.getConfidenceScores),
-        Option(sentencesent.getOpinions).map(sentmap =>
-          sentmap.asScala.toList.map(op =>
-            OpinionV4(getTarget(op.getTarget)
-              , op.getAssessments.asScala.toList.map(assessment =>
-                getAssessment(assessment))))),
-        sentencesent.getOffset,
-        sentencesent.getLength)
-    }
-
-    def getDocumentSentiment(doc: DocumentSentiment): SentimentScoredDocumentV4 = {
-      SentimentScoredDocumentV4(
-        doc.getSentiment.toString,
-        getConfidenceScore(doc.getConfidenceScores),
-        doc.getSentences.asScala.toList.map(sentenceSentiment =>
-          getSentenceSentiment(sentenceSentiment)),
-        doc.getWarnings.asScala.toList.map(warnings =>
-          WarningsV4(warnings.getMessage, warnings.getWarningCode.toString)))
-    }
-
-    val sentimentResult = textSentimentResultCollection.map(sentiment => (sentiment.isError) match {
-      case false => Some(getDocumentSentiment(sentiment.getDocumentSentiment))
-      case true => None
-    }).toList
-
-    val error = textSentimentResultCollection.map(sentiment => (sentiment.isError) match {
-      case true => Some(TAErrorV4(sentiment.getError.getErrorCode.toString,
-        sentiment.getError.getMessage, sentiment.getError.getTarget))
-      case false => None
-    }).toList
-
-    val stats = textSentimentResultCollection.map(result => (result.isError) match {
-      case false => Option(result.getStatistics) match {
-        case Some(s) => Some(DocumentStatistics(s.getCharacterCount, s.getTransactionCount))
-        case None => None
-      }
-      case true => None
-    }).toList
-
-    TAResponseV4[SentimentScoredDocumentV4](
-      sentimentResult,
-      error,
-      stats,
-      Some(resultCollection.getModelVersion))
-  }
-  override def outputSchema: StructType = SentimentResponseV4.schema
-}
+package com.microsoft.ml.spark.cognitive
```

Comment:
Can we make sure the previous codes still present? Only submit the changes you modified?